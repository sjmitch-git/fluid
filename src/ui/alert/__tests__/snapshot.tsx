import { render } from '@testing-library/react'
import { Alert } from '..'

it('renders Alert unchanged', () => {
	const { container } = render(
		<Alert
			status='error'
			message="<p><strong>Ugh umami</strong> bacon butcher <a href='#'>pickled kogi</a> stumptown, <em>subway tile</em> gatekeep flexitarian.</p>"
		/>
	)

	expect(container).toMatchSnapshot()
})
