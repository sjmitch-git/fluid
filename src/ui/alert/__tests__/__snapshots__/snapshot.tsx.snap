// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Alert unchanged 1`] = `
<div>
  <blockquote
    class="alert group relative  text-base px-4 py-1 border-error bg-[var(--error-color-25)] outline-error text-dark border-s-8"
    data-testid="alert"
    role="alert"
  >
    <div
      class="text-size-inherit mb-[1em] mt-[1em]"
    >
      <p>
        <strong>
          Ugh umami
        </strong>
         bacon butcher 
        <a
          href="#"
        >
          pickled kogi
        </a>
         stumptown, 
        <em>
          subway tile
        </em>
         gatekeep flexitarian.
      </p>
    </div>
  </blockquote>
</div>
`;
